contrib/kubespray/readme.md

```diff
@@ -1,456 +1,35 @@
-      ansible_ssh_pass: "your-password-here"
-      ansible_ssh_extra_args: '-o StrictHostKeyChecking=no'
-    node3:
-      ip: x.x.x.39
-      access_ip: x.x.x.39
-      ansible_host: x.x.x.39
-      ansible_ssh_pass: "your-password-here"
-      ansible_ssh_extra_args: '-o StrictHostKeyChecking=no'
-    node4:
-      ip: x.x.x.40
-      access_ip: x.x.x.40
-      ansible_host: x.x.x.40
-      ansible_ssh_pass: "your-password-here"
-      ansible_ssh_extra_args: '-o StrictHostKeyChecking=no'
-    node5:
-      ip: x.x.x.41
-      access_ip: x.x.x.41
-      ansible_host: x.x.x.41
-      ansible_ssh_pass: "your-password-here"
-      ansible_ssh_extra_args: '-o StrictHostKeyChecking=no'
-    node6:
-      ip: x.x.x.42
-      access_ip: x.x.x.42
-      ansible_host: x.x.x.42
-      ansible_ssh_pass: "your-password-here"
-      ansible_ssh_extra_args: '-o StrictHostKeyChecking=no'
-    node7:
-      ip: x.x.x.43
-      access_ip: x.x.x.43
-      ansible_host: x.x.x.43
-      ansible_ssh_pass: "Your-Password"
-      ansible_ssh_extra_args: '-o StrictHostKeyChecking=no'
-    node8:
-      ip: x.x.x.44
-      access_ip: x.x.x.44
-      ansible_host: x.x.x.44
-      ansible_ssh_pass: "Your-Password"
-      ansible_ssh_extra_args: '-o StrictHostKeyChecking=no'
-    node9:
-      ip: x.x.x.45
-      access_ip: x.x.x.45
-      ansible_host: x.x.x.45
-      ansible_ssh_pass: "Your-Password"
-      ansible_ssh_extra_args: '-o StrictHostKeyChecking=no'
-  children:
-    kube-master:
-      hosts:
-        node1:
-        node2:
-        node3:
-    kube-node:
-      hosts:
-        node1:
-        node2:
-        node3:
-        node4:
-        node5:
-        node6:
-        node7:
-        node8:
-        node9:
-    etcd:
-      hosts:
-        node1:
-        node2:
-        node3:
-    k8s-cluster:
-      children:
-        kube-node:
-        kube-master:
-    calico-rr:
-      hosts: {}
-```
-
-- Add the new k8s worker node into the cluster with the following command
-
-```bash
-
-cd kubespray/
-
-ansible-playbook -i inventory/mycluster/hosts.yml upgrade-cluster.yml --become --become-user=root  --limit=node7,node8,node9 -e "@inventory/mycluster/openpai.yml"
-
-```
-
-
-#### After deployment
-
-###### Recover backup data (namespaced_secret) from the backup data after setuping k8s with kubespray.
-
-If you backup your data from the k8s deployed by openpai, you could recover the data to a specified namespace after the k8s deployment of kubespray done.
-
-```
-cd pai/contrib/kubespray
-
-# Before backup, please ensure there are datas in the namespaces.
-# with the command kubectl get secrets -n namespace-name
-python3 namespace_secret_recover.py -n pai-user-v2 -i pai-user-v2
-python3 namespace_secret_recover.py -n pai-group -i pai-group
-python3 namespace_secret_recover.py -n pai-storage -i pai-storage
-python3 namespace_secret_recover.py -n pai-user-token -i pai-user-token
-```
\ No newline at end of file
+### Deploy Kubernetes with kubepsray
+
```

Comment:
- you might want to add a reference link to kubespray (typo in the title ^) and why we are using it.

- suggest to explain which parts are general openpai setup preparations not related to kubespray.

- please also let users know that we do have non-kubespray installation option and add a reference link for it.

- better also mention our usage of kubespray is based on which version

Reply:
Done